other/prepend-image-registry/prepend-image-registry.yaml

```diff
@@ -34,7 +34,7 @@ spec:
         - UPDATE
     mutate:
       foreach:
-      - list: "request.object.spec.containers"
+      - list: "request.object.spec.[ephemeralContainers, initContainers, containers][]"
```

Comment:
This doesn't make sense when there are multiple loops. May need to add a third loop just for ephemeralContainers and change this back.